### PR TITLE
chore: remove CompatGeckoDesktop

### LINKED ISSUE #8965: chore: remove CompatGeckoDesktop
There are still a couple of instances of the `CompatGeckoDesktop` macro. I will do the final bit of cleanup.

### PR DESCRIPTION
**Pull request by schalkneethling**
Remove final instances of the `CompatGeckoDesktop` macro.

fix #8965

### CODE REVIEW

File: files/ja/orphaned/web/javascript/reference/global_objects/asyncfunction/prototype/index.md [hunk 1]
```diff
@@ -27,55 +27,7 @@ original_slug: Web/JavaScript/Reference/Global_Objects/AsyncFunction/prototype
 
 ## ブラウザー実装状況
 
-{{CompatibilityTable}}
```
Comment (schalkneethling) [inline files/ja/orphaned/web/javascript/reference/global_objects/asyncfunction/prototype/index.md:30]: This also removed the deprecated `CompatibilityTable` macro.

Comment (schalkneethling) [reply files/ja/orphaned/web/javascript/reference/global_objects/asyncfunction/prototype/index.md:30]: Although, as this is in an orphaned directory we could just delete the file?

File: files/pt-br/conflicting/web/api/animationevent/animationevent/index.html [hunk 2]
```diff
@@ -72,59 +72,7 @@ <h2 id="Especificações">Especificações</h2>
 
 <h2 id="Browser_compatibility">Compatibilidade com navegadores</h2>
 
-<p>{{ CompatibilityTable }}</p>
```
Comment (schalkneethling) [inline files/pt-br/conflicting/web/api/animationevent/animationevent/index.html:75]: Curious about the intent of the `conflicting` folder. I believe files in a folder called `orphaned` is meant to be deleted as they no longer exist in the English content. What is the utility of the `conflicting` folder? Is this documented somewhere? Thanks!

File: files/pt-br/orphaned/web/api/navigatorlanguage/index.html [hunk 3]
```diff
@@ -58,89 +58,7 @@ <h2 id="Specifications">Specifications</h2>
 
 <h2 id="Browser_compatibility">Compatibilidade com navegadores</h2>
 
-<p>{{ CompatibilityTable() }}</p>
```
Comment (schalkneethling) [inline files/pt-br/orphaned/web/api/navigatorlanguage/index.html:61]: Could we just delete this folder/file in the `orphaned` directory?

File: files/ru/web/api/web_workers_api/functions_and_classes_available_to_workers/index.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
+---
+title: Функции и классы доступные для Web Workers
+slug: Web/API/Web_Workers_API/Functions_and_classes_available_to_workers
+translation_of: Web/API/Web_Workers_API/Functions_and_classes_available_to_workers
+---
+
+{{DefaultAPISidebar("Web Workers API") }}
```
Comment (schalkneethling) [inline files/ru/web/api/web_workers_api/functions_and_classes_available_to_workers/index.md:7]: There was very little left of this file after cleaning up the macro usage, so I synced it with English and converted it to Markdown. I hope that is ok.

**Review by mfuji09 — APPROVED** [review]
Approve for `ja` file.

File: files/ja/orphaned/web/javascript/reference/global_objects/asyncfunction/prototype/index.md (hunk 1 above)
Comment (mfuji09) [reply files/ja/orphaned/web/javascript/reference/global_objects/asyncfunction/prototype/index.md:30]: This file can be deleted, and it should be redirected to "Web/JavaScript/Reference/Global_Objects/AsyncFunction".

**Comment by mfuji09** [comment]
By the way, there are several unresolved "chore" PRs. Most of them are created across several languages.
However, PRs across several languages are needed a longer time to be merged, because it have to be reviewed by several lanugage communities. It can cause the PR older and more conflicts.
If you create separate PRs for each languages, they can be merged more quickly. And they can be bound with an issue.

File: files/pt-br/conflicting/web/api/animationevent/animationevent/index.html (hunk 2 above)
Comment (cw118) [reply files/pt-br/conflicting/web/api/animationevent/animationevent/index.html:75]: Files in the `conflicting` folder are usually moved there by the `sync-translated-content` workflow, which tried to move a file to a "new" slug but a page with that slug already exists (this often occurs when a group of pages are deleted on mdn/content and are all redirected to the same page). 

I believe there isn't really documentation on conflicting/orphaned, but I [wrote a gist for this](https://gist.github.com/cw118/00424f18db4476e8b76dd31842ed9bad) some time ago :)

Comment (schalkneethling) [reply files/pt-br/conflicting/web/api/animationevent/animationevent/index.html:75]: Thank you for the feedback, @cw118. Might be worth adding a more detailed explanation to the README or CONTRIBUTING docs?

**Comment by schalkneethling** [comment]
> If you create separate PRs for each language, they can be merged more quickly. And they can be bound with an issue.

That is a great idea! Thank you for the suggestion.

**Review by Graywolf9 — APPROVED** [review]
LGTM for `es` Thank you so much! @schalkneethling

**Review by JuanVqz — APPROVED** [review]

File: files/ja/orphaned/web/javascript/reference/global_objects/asyncfunction/prototype/index.md (hunk 1 above)
Comment (schalkneethling) [reply files/ja/orphaned/web/javascript/reference/global_objects/asyncfunction/prototype/index.md:30]: > This file can be deleted, and it should be redirected to "Web/JavaScript/Reference/Global_Objects/AsyncFunction".

Should we do it as part of this PR or open an issue to do it post landing this one? Also, last I heard the `delete` function of Yari is broken?

File: files/pt-br/conflicting/web/api/animationevent/animationevent/index.html (hunk 2 above)
Comment (schalkneethling) [reply files/pt-br/conflicting/web/api/animationevent/animationevent/index.html:75]: @cw118 are you going to open a PR for this? You can also push to my branch if you want.

Comment (cw118) [reply files/pt-br/conflicting/web/api/animationevent/animationevent/index.html:75]: @schalkneethling Sorry for the late reply, recent weeks have been busy for me :) I'd be happy to work on this, probably in another PR as I think conflicting/orphaned guidelines are not quite in the scope of this one.

**Review by cw118 — APPROVED** [review]
FWIW for now, to use the `delete` command in yari it should work if we checkout https://github.com/mdn/yari/commit/79f3eb93bfa4a3c1a6c9138fe5c350af4538d3f5 (cf. https://github.com/mdn/translated-content/pull/8636#issue-1378432949) :)

File: files/pt-br/conflicting/web/api/animationevent/animationevent/index.html (hunk 2 above)
Comment (schalkneethling) [reply files/pt-br/conflicting/web/api/animationevent/animationevent/index.html:75]: I 100% agree. I assume you will take care of this one as well? https://github.com/mdn/translated-content/pull/8973#discussion_r988342233

File: files/ru/web/api/web_workers_api/functions_and_classes_available_to_workers/index.md (hunk 4 above)
Comment (sashasushko) [inline files/ru/web/api/web_workers_api/functions_and_classes_available_to_workers/index.md:7]:
```suggestion
{{DefaultAPISidebar("Web Workers API")}}
```

**Review by sashasushko — APPROVED** [review]
Thanks from @mdn/yari-content-ru  🙏

**Review by mfuji09 — APPROVED** [review]

**Comment by Graywolf9** [comment]
Here is it only missing approval for `pt-br` @josielrocha ?

**Review by josielrocha — APPROVED** [review]
Hey @Graywolf9!

It looks fine for `pt-BR`, but there is a conflict here for `ru`.

👍🏽

**Comment by Graywolf9** [comment]
Thank you so much for your approval !  @josielrocha 

So now we are with @sashasushko and @schalkneethling

**Review by sashasushko — APPROVED** [review]

**Comment by cw118** [comment]
It looks like all locale teams have approved, let's merge this 🔥

**Pull request merged**